src/main/java/dbProcs/Database.java

```diff
@@ -6,27 +6,27 @@
 
 import org.apache.log4j.Logger;
 
-/**  

```

Comment:
Did you make any changes to Database.java? This just looks like whitespace changes